.editorconfig

```diff
@@ -0,0 +1,5 @@
+[*.{kt,kts}]
+indent_size=4
+continuation_indent_size=4
+insert_final_newline=true
+max_line_length=120
```

Comment:
According to Kotlin style guide this should be 100. https://developer.android.com/kotlin/style-guide#line_wrapping

Reply:
👍 improved in 1485fe3